### ISSUE TITLE
arguments for mdinclude directive

### ISSUE THREAD

**Issue by papakai** [miyakogi/m2r#25]
if was just playing around with things as we run into an issue with multiple headlines (one from .md file and one in .rst file), but we want to keep the .rst headline.

```
.. mdinclude:: ../HISTORY.md
   :lines: 2-
```

This results into
```
home/kai/git/python-sdk/docs/changelog.rst:4: WARNING: Error in "mdinclude" directive:
maximum 1 argument(s) allowed, 3 supplied.

.. mdinclude:: ../HISTORY.md
   :lines: 2-
```

Would be very great if mdinclude could support also some of the arguments that are available on the literalinclude directive. Let me know if you plan to work on this or not.
Thanks a lot :)

**Comment by miyakogi** [comment]
`lines` option of `literalinclude` may be difficult to implement.
But I want to try to implement some options of `include` directive, such as `start-line` and `end-line`.
http://docutils.sourceforge.net/docs/ref/rst/directives.html#include

**Comment by miyakogi** [comment]
Implemented `start-line` and `end-line` options to `mdinclude` directive and released as v0.2.0.
